### ISSUE TITLE
Neon and Web Assembly Support

### ISSUE THREAD

**Issue by mert-kurttutan** [sarah-ek/gemm#3]
Hi,

Are you thinking of adding support for Neon and Wasm architecture anypoint in the road map of this crate ?

**Comment by sarah-ek** [comment]
i think neon support is already there, (not in faer yet, though)
wasm support is a bit of a tricky subject, since wasm doesn't seem to have a way to detect simd at runtime, so it'd have to be compile time dispatch. i could also use a bit of help since i have little experience with neon/wasm

**Comment by mert-kurttutan** [comment]
Yeah, I realize it now, (I must have assumed from trying to run faer rs on a mac machine.
